src/main/config/config_eeprom.c

```diff
@@ -34,13 +34,13 @@
 #include "pg/pg.h"
 #include "fc/config.h"
 
-#include "drivers/system.h"
-
-#ifndef EEPROM_IN_RAM
-extern uint8_t __config_start;   // configured via linker script when building binaries.
-extern uint8_t __config_end;
+#ifdef EEPROM_IN_SDCARD
```

Comment:
Nitpicking. But `EEPROM_ON_SDCARD` sounds less wrong.

Reply:
I had the same dilemma, i went for 'in' for consistency with the other options in the end.

Reply:
There is to that - I guess there is no non-suboptimal solution here. :-|